docs/development.md

```diff
@@ -145,10 +160,11 @@ components in various configurations and is useful for example to develop and do
 
 See a running guide as example [https://rems-dev.rahtiapp.fi/guide](https://rems-dev.rahtiapp.fi/guide).
 
-## Automated accessibility test report
 
-We use [axe](https://www.deque.com/axe/) for automated accessibility tests.
+## Mac OS installation for Apple M1 Chip (Apple Silicon M1)
+
+    1. Java Installation: install Java via Azul, make sure you download the version of [Java for MacOS for ARM architecture](https://www.azul.com/downloads/zulu-community/?os=macos&architecture=arm-64-bit&package=jdk).
```

Comment:
I think you have too much indentation here: github is rendering this as a code block now.